File: easy/2185.py
class Solution(object):
    def prefixCount(self, words, pref):
        """
        :type words: List[str]
        :type pref: str
        :rtype: int
        """
        size = len(pref)
        ret = 0
        for word in words:
            if len(word)<size:
                continue
            else :
                if word[:size] == pref:
                    ret=ret+1
                else:
                    continue
        
        return ret

File: easy/2185.cpp
class Solution {
public:
    bool isContain(string a,string b){
        if(a.size()<b.size()){return false;}
        for(int i=0;i<b.size();i++){
            if(a[i]!=b[i]){return false;}
        }
        return true;
    }
    int prefixCount(vector<string>& words, string pref) {
        int ret=0;
        for(int i=0;i<words.size();i++){
            if(isContain(words[i],pref)){ret++;}
        }
        return ret;
    }
};